ncurses.yaml

```diff
@@ -7,53 +7,46 @@ package:
     - all
   copyright:
     - paths:
-      - "*"
+        - "*"
       attestation: TODO
       license: MIT
   dependencies:
     runtime:
       - ncurses-terminfo-base
-
 secfixes:
   6.3-r0:
     - CVE-2022-29458
-
 environment:
   contents:
-    repositories:
-      - https://packages.wolfi.dev/bootstrap/stage3
-    keyring:
-      - https://packages.wolfi.dev/bootstrap/stage3/wolfi-signing.rsa.pub
     packages:
       - busybox
       - ca-certificates-bundle
       - build-base
       - ncurses
-
 pipeline:
   - uses: fetch
     with:
-#      uri: https://invisible-mirror.net/archives/ncurses/current/ncurses-${{package.version}}-20220820.tgz
+      #      uri: https://invisible-mirror.net/archives/ncurses/current/ncurses-${{package.version}}-20220820.tgz
```

Comment:
Should we remove this?

Reply:
No, it is the upstream source, but the mirror is flaky.